cfme/tests/cloud/test_tenant_quota.py

```diff
@@ -32,13 +33,22 @@ def roottenant(appliance):
 
 
 @pytest.fixture
-def prov_data(vm_name):
+def prov_data(vm_name, template_name):
     return {
-        "catalog": {'vm_name': vm_name},
+        "catalog": {'vm_name': vm_name, 'catalog_name': {'name': template_name}},
         "environment": {'automatic_placement': True},
+        'properties': {'instance_type': partial_match('m1.large')}
```

Comment:
**Required/Question**

Hardcoded flavor match is not desirable, but if we're going to do it it should be m1.small, not large. Does the test require a large sized instance?

Reply:
Talked to psav about this, acceptable - its just moved up from L91 anyway :)